docs/node/daemon/config/genesis.mdx

```diff
@@ -94,37 +115,54 @@ Below is an example of a genesis file with complete configuration:
 
 ```json title="sample genesis.json file"
 {
-  "genesis_time": "2023-08-28T22:24:04.513472Z",
-  "chain_id": "kwil-chain-RQMl2F",
-  "initial_height": "0",
+  "genesis_time": "2024-09-10T22:00:55.651468Z",
+  "chain_id": "kwil-chain-p2rrR6RH",
+  "initial_height": 1,
   "alloc": {
-    "247620d8b3e92ac21b74bbda3e51d59de5b9a210b740bba0d3683579ddf6d8dd": 10000000000000000000000, // validator ed25519 public key
-    "0xc89d42189f0450c2b2c3c61f58ec5d628176a1e7": 10000000000000000000000, // user ethereum address
-  }
+    "247620d8b3e92ac21b74bbda3e51d59de5b9a210b740bba0d3683579ddf6d8dd": 10000000000000000000000,
+    "0xc89d42189f0450c2b2c3c61f58ec5d628176a1e7": 10000000000000000000000
+  },
+  "app_hash": null,
+  "activations": null,
   "consensus_params": {
     "block": {
-      "max_bytes": "22020096",
-      "max_gas": "-1"
+      "max_bytes": 6291456,
+      "max_gas": -1,
+      "abci_max_bytes": false
+    },
+    "evidence": {
+      "max_age_num_blocks": 100000,
+      "max_age_duration": 172800000000000,
+      "max_bytes": 1048576
+    },
+    "version": {
+      "app": 0
     },
     "validator": {
       "pub_key_types": [
         "ed25519"
       ],
-      "join_expiry": "86400"
+      "join_expiry": 14400
     },
-    "version": {
-      "app": "0"
+    "votes": {
+      "vote_expiry": 0,
+      "max_votes_per_tx": 100
+    },
+    "abci": {
+      "vote_extensions_enable_height": 0
+    },
+    "migration": {
+      "start_height": -1,
```

Comment:
Defaults for these should be zero now.  Did you generate this from main or preview?